.gitignore

```diff
@@ -102,3 +102,14 @@ venv.bak/
 
 # mypy
 .mypy_cache/
+
+
+#url_sound
+from selenium import webdriver
+import time
+
+driver = webdriver.Chrome('D:\Новая папка\chromedriver.exe')
```

Comment:
Please add a TODO to move hardcoded driver path to arg parse option and to take it from there if it is specified.